docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/jwt-authentication.md

```diff
@@ -0,0 +1,22 @@
+---
+title: JWT Authenticaiton
```

Comment:
Probably should expand the acronym in the title. In-text references expand the acronym on first use, then use JWT throughout